ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/geometry.view.js

```diff
@@ -78,24 +78,47 @@ const GeometryView = Marionette.ItemView.extend({
       this.stopListening(this.options.clusterCollection)
     }
   },
+  adjustPoints(type, coordinates) {
+    coordinates.forEach((coord, index) => {
+      if(index + 1 < coordinates.length) {
+        let negative = false
```

Comment:
what do you need this `negative` local variable for? I think the logic would still be the same if you got rid of it